README.rst

```diff
@@ -1,5 +1,50 @@
 # masabot
-An automated discord bot for handling typical tasks
+An automated discord bot for handling typical tasks.
+
+## Dev Setup
+Masabot requires python 3. If you don't have it, install it.
+
+To install `masabot`, first clone the repo to the local system::
+
+    git clone git@github.com:moe-serifu-circle/masabot.git
+
+Masabot is currently a long-running foreground process; to prevent accidental disconnection ighly-recommended
+
+Masabot has a `setup.py`, so that can be used to install all library dependencies, but this should be done in a virtual
```

Comment:
Have you decided against using pipenv? Pipenv manages a virtual environment for you. Just install pipenv then do `pipenv install; pipenv shell`